ocsp_status_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"crypto"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+)
+
+const (
+	GoodResp = "./testdata/resps/oldfbresp"
+	GoodCert = "./testdata/certs/google.der"
+	ExpiredCert = "./testdata/certs/expiredcert.der"
+	NoIssuingURLCert = "./testdata/certs/rootcert.der"
+	GoodURL = "google.com:443"
+	RevokedURL = "revoked.grc.com:443"
+	BadURL = "blah.blah.blah"
+)
+
+// TestMain sets up the testing framework
+func TestMain(m *testing.M) {
```

Comment:
A general testing tip: keep self-documentation in mind when naming tests. Focus on the job / role / the test performs and not the func / in / outputs it test. Another way to think about unit test names is that they should reflect your low level requirements or contract. 

For example: instead of TestRegisterUser() which corresponds to a function called RegisterUser(), it might be more informative to test TestUserExists() if one of your low level requirements is to make sure after you register the user, the user is reflected in your look up query. 

Moving onto this test file, let's pick the function  ```func TestCheckFromFile``` has the comment "TestCheckFromFile tests checkFromFile", it's not clear what CheckFromFile actually does and this does not self-document well. In CheckFromFile's case it reads a OCSP response, parses and lints the functions. This function does 3 things and is probably more of an integration test rather than a unit test. I would probably start by parsing the various sub-requirements you have for ParseAndLint and test these, then come up with behavioral requirements you need for CheckFromFile and name the tests accordingly.

Hope to see you apply this naming philosophy to the tests you write, we can talk more during 1:1s too :)

Reply:
Thank you for the detailed comment! I've implemented a new testing structure in https://github.com/googleinterns/ocsp-response-linter/pull/13. I've changed my testing comments to be more descriptive there, and in addition, I've leveraged the subtest structure to give an explanation to each test I'm conducting. 

I'm going to merge this branch into master to avoid merge conflicts and too many layers of branch abstraction (I don't want a branch of a branch of a branch of a branch haha).